common/content-api.js

```diff
@@ -8,59 +10,37 @@ const map = require('lodash/fp/map');
 const pick = require('lodash/pick');
 const sortBy = require('lodash/fp/sortBy');
 
-const request = require('request-promise-native');
-const querystring = require('querystring');
-
 const logger = require('./logger');
+const { sanitiseUrlPath } = require('./urls');
+const { CONTENT_API_URL } = require('./secrets');
 
 const getAttrs = response => get('data.attributes')(response);
 const mapAttrs = response => map('attributes')(response.data);
 
-const { sanitiseUrlPath } = require('./urls');
-let { CONTENT_API_URL } = require('./secrets');
-
-function fetch(urlPath, options) {
-    logger.debug(
-        `Fetching ${CONTENT_API_URL}${urlPath}${
-            options && options.qs ? '?' + querystring.stringify(options.qs) : ''
-        }`
-    );
-
-    const defaults = {
-        url: `${CONTENT_API_URL}${urlPath}`,
-        json: true
-    };
-    const params = Object.assign({}, defaults, options);
-    return request(params);
-}
-
-/**
- * Fetch all locales for a given url path
- * Usage:
- * ```
- * fetchAllLocales(reqLocale => {
- *   return `/v1/${reqLocale}/funding-programmes`
- * }).then(responses => ...)
- * ```
- */
-function fetchAllLocales(toUrlPathFn, options = {}) {
-    const urlPaths = ['en', 'cy'].map(toUrlPathFn);
-    const promises = urlPaths.map(urlPath => fetch(urlPath, options));
-    return Promise.all(promises);
-}
+const queryContentApi = got.extend({
```

Comment:
The `extend` syntax and hooks for this library makes for a really nice API. No more confusing `fetch` wrapper just so we can log what content API requests are made locally.